packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -72,7 +75,11 @@
                 <div class="oc-flex oc-flex-between oc-flex-middle">
                   <div class="oc-flex oc-flex-middle">
                     <oc-icon class="oc-mr-s" name="layout-grid" />
-                    <router-link class="space-name oc-text-left" :to="getSpaceProjectRoute(space)">
+                    <router-link
```

Comment:
Please use an oc-button with clickhandler if the space is disabled, i.e. `<oc-button v-if="space.disabled"` and the `<router-link v-else`. Reasoning: for accessibility reasons links should not have click handlers.

Reply:
done